python/sglang/srt/models/torch_native_llama.py

```diff
@@ -208,17 +196,11 @@ def __init__(
         self.qkv_proj.total_num_heads = self.total_num_heads
         self.qkv_proj.head_size = self.head_dim
         self.qkv_proj.total_num_kv_heads = self.total_num_kv_heads
-        self.qkv_proj.num_heads = self.total_num_heads
-        self.qkv_proj.num_kv_heads = self.total_num_kv_heads
+        self.qkv_proj.num_heads = self.num_heads
+        self.qkv_proj.num_kv_heads = self.num_kv_heads
```

Comment:
To differentiate from `qkv_proj.total_num_heads` above

Reply:
so this is the sharded num_heads and num_kv_heads

Reply:
The differentiation is not needed now (removed)